Updating preview statuses for adapters (#8993)
fishtown-analytics/docs.getdbt.com: website/docs/docs/local/connect-data-platform/databricks-setup.md

<VersionBlock firstVersion="2.0">

# Connect Databricks to Fusion <Lifecycle status='private_preview' />

:::note private preview

> The Databricks adapter is supported on the <Constant name="fusion_engine" />, but due to it's status as `Private Preview` it might not register as a supported adapter in some workflows in the <Constant name="dbt_platform" /> interface.

Reviewer: ```suggestion
The Databricks adapter is supported on the <Constant name="fusion_engine" />, but due to its status as `Private Preview`, you might not see it as a supported adapter in some <Constant name="dbt_platform" /> interface workflows.
```
